process/block/postprocess/feeHandler.go

```diff
@@ -65,11 +65,17 @@ func (f *feeHandler) ProcessTransactionFee(cost *big.Int, devFee *big.Int, txHas
 	}
 
 	f.mut.Lock()
-	fee := &feeData{
-		cost:   big.NewInt(0).Set(cost),
-		devFee: big.NewInt(0).Set(devFee),
+	fee, ok := f.mapHashFee[string(txHash)]
+	if !ok {
+		fee = &feeData{
+			cost:   big.NewInt(0),
+			devFee: big.NewInt(0),
+		}
 	}
 
+	fee.cost.Add(fee.cost, cost)
```

Comment:
:1st_place_medal:

Reply:
could the same txHash appear 2 times? If so, RevertFees does the revert only 1 time. Line 97.